submissions/description/insufficient_security_configurability/weak_password_reset_implementation/token_is_not_invalidated_after_login/recommendations.md

```diff
@@ -1,11 +1,11 @@
 # Recommendation(s)
 
-The password reset token should be invalidated after a user successfully resets their password and logs into the application. Overall, the password reset implementation should conform to the following guidelines:
+It is best practice to invalidate the password reset token after after a user successfully resets their password and logs into the application. Overall, the password reset implementation should conform to the following guidelines:
 
-A secure password policy should be in place for the user to create a strong new password
-Password reset tokens should be long to protect against brute force guessing attacks, linked to an individual, invalidated after use, and have a short expiry time
-Passwords should be stored and transmitted securely
-Once a user’s password has been reset, they should be prompted to login in again through the usual login portal and not automatically signed in
+- A secure password policy should be in place for the user to create a strong new password
+- Password reset tokens should be long to protect against brute force guessinging attacks, linked to an individual, invalidated after use, and have a short expiry time
```

Comment:
"guessinging"?

Reply:
Guessing that I didn't have enough coffee that day